vnext/Microsoft.ReactNative/Modules/I18nManagerModule.h

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 #pragma once
 
+#include "../../codegen/NativeI18nManagerSpec.g.h"
```

Comment:
Are other inclusions of the spec done as relative imports?

Reply:
Some other modules #include the spec file in this way too.